src/tag/classes.scss

```diff
@@ -6,7 +6,19 @@
   @include iui-tag;
 }
 
-a.iui-tag {
+.iui-tag-basic {
+  @include iui-tag-basic;
+}
+
+.iui-tag-label {
+  @include iui-tag-label;
+}
+
+.iui-tag-button {
+  @include iui-tag-button;
+}
+
+a.iui-tag-basic {
```

Comment:
Focus state looks weird:
![image](https://user-images.githubusercontent.com/36186912/161498458-d9760fcf-6726-45f1-87da-034eddf6069c.png)

Reply:
Yeah, but it was same before :D
Might have been broken with focus refactors @FlyersPh9  ?

Reply:
I think the top/bottom clipping was present with the old focus styling as well, it just wasn't as noticeable.  I tried using `overflow-x` but that didn't work as expected.  Maybe decrease the height of the anchor and add a padding to the container so the focus doesn't go out of bounds?

Reply:
I changed offset for anchor focus.
@FlyersPh9 wanna make this change in your anchor pr maybe?

Reply:
Looks fine to me, I'll make the anchor PR match in style.